make_profiler/report/index.html

```diff
@@ -0,0 +1,23 @@
+<html>
+
+<head>
+    <title>Kontur Make Profiler Report</title>
+    <link rel="icon" href="favicon.png" type="image/x-icon">
+    <link rel="stylesheet" href='report/style.css'>
+    <link href='https://fonts.googleapis.com/css?family=IBM Plex Sans Condensed' rel='stylesheet'>
+    <script src='report/script_u.js' type="text/javascript"> </script>
+
+    <script>
```

Comment:
suggest to simplify it a bit: 
1 just load "report/sorttable.js"; before 'report/script_u.js'

```
<script src="report/sorttable.js"" type="text/javascript"></script>
<script src="report/script_u.js" type="text/javascript"></script>
```
and to init sortable script in script.js line 107 

```
    const status = document.getElementById("status");
    status.innerHTML = pipelineTable + statusTable;
    sorttable.makeSortable(status.querySelector('.sortable'));
```
got from here (https://www.kryogenix.org/code/browser/sorttable/#ajaxtables)

Reply:
BTW, to simplify it even more you can just do 
`uglifyjs sorttable.js script.js -o minified.js --compress --mangle`
and include only minified.js  in index html  (script_u.js previously)
it will be sorttable.js + script.js  minified uglified inside